__tests__/components/App.test.js

```diff
@@ -4,14 +4,18 @@ import 'jsdom-global/register'
 import React from 'react'
 import { mount, shallow } from "enzyme"
 import { MemoryRouter } from "react-router"
-import RootContainer from '../../src/components/RootContainer'
+import { Provider } from 'react-redux'
 import App from '../../src/components/App'
-import Config from '../../src/Config'
+import LoginPanel from '../../src/components/LoginPanel'
 import HomePage from '../../src/components/HomePage'
 import Editor from '../../src/components/editor/Editor'
 import Browse from '../../src/components/editor/Browse'
+import CanvasMenu from '../../src/components/CanvasMenu'
 import Footer from '../../src/components/Footer'
-import ImportResourceTemplate from "../../src/components/editor/ImportResourceTemplate";
+import ImportResourceTemplate from '../../src/components/editor/ImportResourceTemplate';
```

Comment:
Trivial: You didn't add it, but we can ditch the line-ending semi-colon, which is inconsistent w/ the above `import`s.